libs/shared/utils/src/lib/validateClientId.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Validates that the client id is prefixed with the tenant and that it matches the regex
+ * Value can only contain alphanumeric characters, hyphens, underscores, periods and forward slashes.
+ * @returns true if valid, false otherwise
+ */
+export const validateClientId = ({
```

Comment:
This is quite auth related. Wonder how it fits into the naming/folder convention.

`libs/auth/shared`?

Reply:
I found it hard to find a correct place for this. Creating new lib is a good idea!